scripts/dist.sh

```diff
@@ -1,25 +1,23 @@
 #!/bin/sh
 #
 # Script to create the distribution tarball.
+
 set -e
 
 SRC="$PWD"
 
 # Build the frontend.
 cd "$SRC/frontend"
 pnpm install
-
-TMP="$(mktemp -d)"
-export DESTDIR="$TMP/xdcstore"
-mkdir "$DESTDIR"
-
 pnpm build
 
 # Build the backend.
 cd "$SRC"
-
 cargo build --target x86_64-unknown-linux-musl --release
 
+TMP="$(mktemp -d)"
+DESTDIR="$TMP/xdcstore"
```

Comment:
it is better now to modify this, in the zip don't include a folder just to have a binary inside, zip the binary directly in the root